scripts/build-java.sh

```diff
@@ -16,7 +16,7 @@ for pomFile in $(find $scriptdir/../java -name pom.xml); do
         cd $(dirname $pomFile)
         if [[ -f DO_NOT_AUTOTEST ]]; then exit 0; fi
 
-        mvn compile test
+        mvn -q compile test
```

Comment:
What's the motivation here? The output isn't _that_ noisy, and part of me always wants just a bit more info when diagnosing a problem. That being said, this is the first time I'm looking at this build logs, so happy to defer to your judgment.

Reply:
Well I just had to spend ~10 minutes scrolling down past maven output (because of the verbosity of it's downloads) in Github Actions to find the test that *really* failed... It was not pleasant.